vllm/config.py

```diff
@@ -1114,6 +1114,14 @@ def get_image_input_enum_type(cls, value: str) -> ImageInputType:
                              f"Expecting to choose from "
                              f"{[x.name for x in cls.ImageInputType]}.") from e
 
+    def get_image_token_text(
```

Comment:
the computed value can be cached, but it is also super simple to recompute.

Reply:
The way `VisionLanguageConfig` was designed in #3042 made this call have to depend on the tokenizer, which's less clean. I left a TODO on adding `@cached_property` on this call once we refactor `VisionLanguageConfig`